code/debugconsole/consoleparse.cpp

```diff
@@ -456,105 +450,6 @@ bool dc_maybe_stuff_string_white(SCP_string &str)
 	}
 }
 
-void dc_required_string(char *pstr)
```

Comment:
These are also library functions so they should not be removed.

Reply:
done